cmd/harmony/config/config_test.go

```diff
@@ -0,0 +1,302 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	nodeconfig "github.com/harmony-one/harmony/internal/configs/node"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	trueBool = true
+)
+
+type testCfgOpt func(config *HarmonyConfig)
+
+func makeTestConfig(nt nodeconfig.NetworkType, opt testCfgOpt) HarmonyConfig {
+	cfg := GetDefaultHmyConfigCopy(nt)
+	if opt != nil {
+		opt(&cfg)
+	}
+	return cfg
+}
+
+var testBaseDir = ".testdata"
+
+func init() {
+	if _, err := os.Stat(testBaseDir); os.IsNotExist(err) {
+		os.MkdirAll(testBaseDir, 0777)
+	}
+}
+
+func TestV1_0_0Config(t *testing.T) {
+	testConfig := `Version = "1.0.4"
```

Comment:
Would you please make this data a file and add it to test data folder? This will help with future updates.